src/arithmetic/arithmetic_expression.c

```diff
@@ -549,14 +510,19 @@ void AR_EXP_CollectEntities(AR_ExpNode *root, rax *aliases) {
 
 void AR_EXP_CollectAttributes(AR_ExpNode *root, rax *attributes) {
 	if(root->type == AR_EXP_OP) {
+		if(RG_STRCMP(root->op.func_name, "property") == 0) {
```

Comment:
I was concerned that this would default to the `strcmp` case, in which case I was going to propose using a const string, but some local testing suggests that's unnecessary.